package.json

```diff
@@ -5,7 +5,8 @@
   "main": "dist/lib/rule-finder.js",
   "scripts": {
     "cover": "nyc --reporter=lcov --reporter=text npm test",
-    "lint": "xo",
+    "lint": "eslint --ext .js,.json .",
+    "lint-fix": "npm run lint -- --fix",
```

Comment:
not sure this script is needed at all; you can always stick a `--` on the end of the `lint` script

Reply:
That was a lazy copy/paste.